delete-publication-from-scopus/src/test/java/no/sikt/nva/scopus/ScopusDeleteHandlerTest.java

```diff
@@ -0,0 +1,133 @@
+    private S3Driver s3Driver;
+    private ScopusDeleteHandler scopusDeleteHandler;
+    private FakeEventBridgeClient fakeEventBridgeClient;
+
+
+    @BeforeEach
+    public void init() {
+        s3Client = new FakeS3Client();
+        s3Driver = new S3Driver(s3Client, "ignoredValue");
+        fakeEventBridgeClient = new FakeEventBridgeClient();
+        scopusDeleteHandler = new ScopusDeleteHandler(s3Client, fakeEventBridgeClient);
+    }
+
+    @Test
+    void shouldEmitEventWithSingleScopusIdentifierWhenInputFileContainsSingleScopusIdentifier() throws IOException {
+        var scopusFile = IoUtils.stringFromResources(Path.of(DELETE_SINGLE_IDENTIFIER_FILE));
+        var uri = s3Driver.insertFile(UnixPath.of(randomString()), scopusFile);
+        var s3Event = createS3Event(uri);
+        scopusDeleteHandler.handleRequest(s3Event, CONTEXT);
+        var requestEntries = fakeEventBridgeClient.getRequestEntries();
+        assertThat(requestEntries.size(), is(equalTo(1)));
+        var eventBodyJson = requestEntries.get(0).detail();
+        var actualRequestEntryBody =
+                JsonUtils.dtoObjectMapper.readValue(eventBodyJson, ScopusDeleteEventBody.class);
+        var expectedRequestEntryBody = new ScopusDeleteEventBody(HARDCODED_SCOPUS_IDENTIFIER_IN_DELETE_SINGLE_FILE);
+        assertThat(actualRequestEntryBody, is(equalTo(expectedRequestEntryBody)));
+    }
+
+    @Test
+    void shouldEmitEventsWithScopusIdentifiersWhenInputFileContainsMultipleScopusIdentifiers() throws IOException {
+        var scopusFile = IoUtils.stringFromResources(Path.of(DELETE_MULTIPLE_IDENTIFIERS_FILE));
+        var uri = s3Driver.insertFile(UnixPath.of(randomString()), scopusFile);
+        var s3Event = createS3Event(uri);
+        scopusDeleteHandler.handleRequest(s3Event, CONTEXT);
+        var requestEntries = fakeEventBridgeClient.getRequestEntries();
+        assertThat(requestEntries.size(), is(equalTo(HARDCODED_SCOPUS_IDENTIFIERS_IN_DELETE_MULTIPLE_FILE.size())));
+        var eventBodyJson = requestEntries.get(0).detail();
+        var actualRequestEntryBody =
+                JsonUtils.dtoObjectMapper.readValue(eventBodyJson, ScopusDeleteEventBody.class);
+        var expectedRequestEntryBody = new ScopusDeleteEventBody(HARDCODED_SCOPUS_IDENTIFIER_IN_DELETE_SINGLE_FILE);
+        assertThat(actualRequestEntryBody, is(equalTo(expectedRequestEntryBody)));
+    }
+
+    @Test
+    void shouldReturnCorrectIdentifiersFromFileContentWhenEventWithS3UriThatPointsToScopusDeleteFile()
+            throws IOException {
+        var scopusFile = IoUtils.stringFromResources(Path.of(DELETE_MULTIPLE_IDENTIFIERS_FILE));
+        var uri = s3Driver.insertFile(UnixPath.of(randomString()), scopusFile);
+        var s3Event = createS3Event(uri);
+        var identifiersToDelete = scopusDeleteHandler.handleRequest(s3Event, CONTEXT);
+        assertThat(identifiersToDelete, is(equalTo(HARDCODED_SCOPUS_IDENTIFIERS_IN_DELETE_MULTIPLE_FILE)));
+    }
```

Comment:
where is the output sent?

Reply:
This test can probably be deleted/replaced, I am not sure what we want the handler to return if anything.